plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -19,10 +19,11 @@ const config = {
 	globalSetup: require.resolve( './global-setup' ),
 	globalTeardown: require.resolve( './global-teardown' ),
 	testDir: 'tests',
-	retries: CI ? 4 : 2,
-	workers: 4,
+	retries: CI ? 2 : 2,
```

Comment:
Not a blocker, but since these values are the same, we can probably remove the ternary operator here